analyzer/src/funTest/kotlin/Yarn2Test.kt

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
commit-msg: Would it make sense to move the reference `[1]` behind the first mention of "Corepack" in the message _body_?